[ASM] - Expander - Enrichment Status and N/A values (3581 & 4820) (#28663)
demisto/content: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_27.md

#### Incident Fields

##### ASM - Enrichment Status

> Added a new field, **ASM - Enrichment Status**, to track the status of types of platforms enrichment has been performed with.

Reviewer: ```suggestion
Added a new field, **ASM - Enrichment Status**, to track the status of types of platform enrichment that has been performed.
```

##### ASM - Dev Check Details

> Update the comment field of the `result` key.

Reviewer: ```suggestion
Updated the comment field of the *result* key.
```

##### Cortex ASM - AWS Enrichment

- Playbook now supports populating the ASM - Enrichment Status field to identify if enrichment was completed successfully or not.
- Updated the use of `N/A` to `n/a` for when various results are not found in the playbook to follow internal conventions.

##### Cortex ASM - Qualys Enrichment

- Playbook now supports populating the ASM - Enrichment Status field to identify if enrichment was completed successfully or not.
- Updated the use of `N/A` to `n/a` for when various results are not found in the playbook to follow internal conventions.

##### Cortex ASM - ServiceNow CMDB Enrichment

Playbook now supports populating the ASM - Enrichment Status field to identify if enrichment was completed successfully or not.

##### Cortex ASM - Azure Enrichment

- Playbook now supports populating the ASM - Enrichment Status field to identify if enrichment was completed successfully or not.
- Updated the use of `N/A` to `n/a` for when various results are not found in the playbook to follow internal conventions.

##### Cortex ASM - Tenable.io Enrichment

> - Playbook now supports populating the ASM - Enrichment Status field to identify if enrichment was completed successfully or not.

Reviewer: ```suggestion
Playbook now supports populating the ASM - Enrichment Status field to identify if enrichment was completed successfully or not.
```
